CHANGELOG.rst

```diff
@@ -2,6 +2,209 @@
 Release Notes
 =============
 
+.. _Release Notes_7.57.0:
+
+7.57.0
+======
+
+.. _Release Notes_7.57.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2024-09-09
+
+- Please refer to the `7.57.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7570>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_7.57.0_Upgrade Notes:
+
+Upgrade Notes
+-------------
+
+- Update cURL to 8.7.1.
+
+- Update OpenSSL from 3.0.14 to 3.3.1.
```

Comment:
We've been asked to clarify this since Windows customers find it confusing when they find older versions of OpenSSL from what's reported here:

```suggestion
- Update OpenSSL from 3.0.14 to 3.3.1 (on Linux and macOS).
```